### DISCUSSION TITLE
엔터프라이즈 애플리케이션 아키텍처 참여 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [ThinkAboutSoftware/AcademicConference#387] [General]
안녕하세요.

이번 싸이클은 쉬어가고 `객체지향의 사실과 오해` 싸이클 때 다시 돌아오겠습니다. 미리 말씀드렸어야 했는데, 책을 읽다 보니 지금은 제가 더 집중해야 할 영역이 있는 것 같다는 생각이 들어서 그렇게 해보려고 합니다.

**Comment by jongfeel** [comment]
알겠습니다.

학교 다시 잘 다니시고 하려는 것도 집중해서 의미있는 시간을 만들어 보면 좋겠습니다.
